metaci/db/management/commands/populate_db.py

```diff
@@ -1,59 +1,71 @@
-from django.core.management.base import BaseCommand
+import random
 
+import factory.random
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand
 from django.db import transaction
-
 from guardian.shortcuts import assign_perm
 
-import factory.random
-
 from metaci import conftest as fact
-from metaci.users.models import User
-from metaci.testresults.models import TestResult
-from django.contrib.auth.models import Group
-
 from metaci.build.models import BuildFlow
-
-import random
+from metaci.cumulusci.models import Org
+from metaci.plan.models import Plan, PlanRepository
+from metaci.repository.models import Repository
+from metaci.testresults.models import TestResult
+from metaci.users.models import User
 
 
 class Command(BaseCommand):
     help = "Populates a testing DB with sample data"
 
-    def handle(self, *args, **options):
-        if self.check_non_destructive():
-            print("Populating test data")
-            # reset_db.Command().handle(*args, **options)
-            # migrate.Command().handle(*args, **options)
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--results", action="store_true", help="Create fake builds and test results"
+        )
 
-            self.create_objs()
-        else:
-            print(
+    def handle(self, *args, **options):
+        if options["results"]:
+            if self.check_non_destructive():
+                print("Populating test data")
+                # reset_db.Command().handle(*args, **options)
+                # migrate.Command().handle(*args, **options)
+
+                self.create_builds_with_test_results()
+            else:
+                print(
+                    """
+                This does not seem to be an empty or test database!
```

Comment:
Salesforce writing guides recommend not using exclamation points in error messages and instructional text. I recommend removing the exclamation mark on this and the next line. 

The [writing style guidelines trail ](https://trailhead.salesforce.com/content/learn/modules/writing_style/writing_style_guidelines)says this:

> - Use exclamation points only to be encouraging or to generate excitement.
> - Don’t use exclamation points in error messages, confirmation messages, or instructional text.

Reply:
Good to know, though so far the only real audience for this command is Paul.